engine/src/main/java/org/pentaho/di/core/util/ConnectionUtil.java

```diff
@@ -24,10 +24,8 @@
 
 import org.pentaho.di.base.AbstractMeta;
 import org.pentaho.di.connections.ConnectionManager;
+import org.pentaho.di.core.osgi.api.MetastoreLocatorOsgi;
```

Comment:
Is this import actually being used?

Reply:
Good catch. Looks like it was used in an interim commit. Will remove.